docker-compose.yml

```diff
@@ -467,6 +467,26 @@ services:
     ports:
       - "15000:15000"
 
+  #
+  # The second version of "live replay" server
+  #
+  faf-aio-replayserver:
+    container_name: faf-aio-replayserver
+    build: git://github.com/FAForever/faf-aio-replayserver
+    image: faforever/faf-aio-replayserver:0.1
```

Comment:
Once 0.1 is published on docker hub, we're good to go

Reply:
https://hub.docker.com/r/faforever/faf-aio-replayserver/tags/